images/openwisp_base/Dockerfile

```diff
@@ -8,93 +8,86 @@ FROM python:3.10-slim-bullseye AS system
 RUN apt-get update && \
     apt-get install --yes --no-install-recommends \
     libcairo2 apt-utils libpangocairo-1.0-0 \
-    gdal-bin gettext fping && \
+    gdal-bin gettext fping nano && \
```

Comment:
Why do we need an editor in production images?
Did you add this during testing?

Reply:
Well my reasoning was that since the containers are running unprivileged then getting an editor there is not very straightforward in case something breaks in production. 80kB or how much it weights sounds like a meaningful sacrifice. Given that the image itself has over 1.1GB